src/clarity-angular/tabs/vtabs-nav-group.ts

```diff
@@ -0,0 +1,25 @@
+
+import {Component} from "@angular/core";
+
+/**
+ * @class VTabsNavGroup
+ *
+ * @description
+ * A child component of VTabs that is a container for a group of nav links.
+ * An optional label can be passed in to give the group a title.
+ * clr-vtabs-nav-links are projected to the VTabsNavLink class.
+ *
+ * Style is set in the template, not dynamically.
+ */
+@Component({
+    selector: "clr-vtabs-nav-group",
+    template: `
+        <div class="nav-group">
+            <ng-content select="label"></ng-content>
```

Comment:
@hippee-lee, @mathisscott: Oh I didn't know that the wizard was doing the same thing. I might be wrong here then. But I remember that dropdowns had an issue because we used to query by class `.dropdown-item` and `.dropdown-toggle`. We introduced a breaking change to replace them by directives because of the number of issues it was causing in the demo app. But that definitely might a case just with dropdowns. 
We should check with @youdz on whats the right approach to handle such cases. But irrespective of that querying by `label` i think can be improved to query by `.nav-group-header` class just to be more specific and I say that because if someone use a `span` instead of a `label` this might break.